org.adempiere.base/src/org/compiere/process/DocumentEngine.java

```diff
@@ -379,10 +379,47 @@ public boolean processIt (String action)
 		}
 		if (ACTION_ReActivate.equals(m_action))
 			return reActivateIt();
-		if (ACTION_Reverse_Accrual.equals(m_action))
-			return reverseAccrualIt();
-		if (ACTION_Reverse_Correct.equals(m_action))
-			return reverseCorrectIt();
+		if (ACTION_Reverse_Accrual.equals(m_action) || ACTION_Reverse_Correct.equals(m_action))
+		{
+			boolean ok = false;
+			if (ACTION_Reverse_Accrual.equals(m_action))
+				ok = reverseAccrualIt();
+			else if (ACTION_Reverse_Correct.equals(m_action))
+				ok = reverseCorrectIt();
+			
+			if (m_document != null && ok)
+			{
+				// PostProcess documents
+				List<PO> docsPostProcess = new ArrayList<PO>();
+				if (m_document instanceof IDocsPostProcess) {
+					docsPostProcess = ((IDocsPostProcess) m_document).getDocsPostProcess(); 
+				}

```

Comment:
if we move the instanceof check to the outer if condition (like what I've suggested above), this block can be simplified to `List<PO> docsPostProcess = ((IDocsPostProcess) m_document).getDocsPostProcess();`